modules/files.nix

```diff
@@ -256,9 +256,9 @@ in
         }
         declare -A changedFiles
       '' + concatMapStrings (v: ''
-        _cmp "${sourceStorePath v}" "${homeDirectory}/${v.target}" \
-          && changedFiles["${v.target}"]=0 \
-          || changedFiles["${v.target}"]=1
+        _cmp --quiet ${escapeShellArg (sourceStorePath v)} ${escapeShellArg homeDirectory}/${escapeShellArg v.target} \
```

Comment:
Hmm, I don't think `_cmp` takes a `--quiet` argument.

Reply:
Sorry, this was an improper merge.